chain-impl-mockchain/src/evm/mod.rs

```diff
@@ -16,7 +16,36 @@ use crate::{
 };
 
 #[cfg(feature = "evm")]
-pub use chain_evm::machine::{BlockGasLimit, Config, Environment, GasPrice};
+pub use chain_evm::machine::{
+    BlockGasLimit, BlockTimestamp, ChainId, Config, Environment, GasPrice,
+};
+
+/// Static parameters for EVM
```

Comment:
this is static but contains `BlockEpoch` which is a dynamic parameter, have I missed something?

Reply:
Just a typo, thanks for pointing it out.